packages/lab/src/slider/internal/SliderMarkLabels.tsx

```diff
@@ -18,6 +21,14 @@ export interface SliderMarkLabelsProps {
 
 export function SliderMarkLabels(props: SliderMarkLabelsProps) {
   const { min, max, marks } = props;
+
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-slider-handle",
```

Comment:
testid is optional but maybe this should be "salt-slider-mark-labels"? Same applies to a few of the other imports